### PR TITLE
Works with any filename

### PR DESCRIPTION
**Pull request by mertbulan**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -4,7 +4,7 @@ You can create an app in Heroku with Crystal's buildpack by running the
 following command:
 
 ```bash
-$ heroku create myapp --buildpack https://github.com/zamith/heroku-buildpack-crystal.git
+$ heroku create myapp --buildpack https://github.com/mertbulan/heroku-buildpack-crystal.git
```
Comment (zamith) [inline README.markdown]: Why change this?

**Comment by zamith** [comment]
I don't really understand why you would want this. It is normal (good?) for any language to have conventions. Can you please elaborate on why you would want this change?

**Comment by mertbulan** [comment]
It was a mistake. I didn't realize the changing in pull request. I change it because maybe you don't merge it. If some people want to use my repo, maybe they confuse the link.

**Comment by zamith** [comment]
That's fine. But can you please elaborate on why you would want this feature, either as a comment or in the PR's description, please.

**Comment by mertbulan** [comment]
Because when you create an app with "crystal init app your_app", it creates files with app's name. Your version works with app.cr, if I want to deploy to Heroku I must fork buildpack for every app. In my verison, I don't need create buildpack for every app.

**Comment by f** [comment]
@zamith I actually agree with this approach. Since Crystal's convention is `something.cr`, renaming it to `app.cr` to work for Heroku sounds not good.

I see only problem with `for f in *` part, it looks for all the files I think, `for f in *.cr` would be more consistent. @mertbulan

**Comment by mertbulan** [comment]
@f It looks for all the files because executable file doesn't have file type. It looks all the files and finds the executable one.

**Comment by f** [comment]
Ah sorry, you're right.

**Comment by zamith** [comment]
Right, I get it, what about #6, doesn't it solve this problem in a better way? What do you think of moving that one along in favor of this?

**Comment by mertbulan** [comment]
This way or that way, doesn't matter. I think one of them should be merged.

**Comment by zamith** [comment]
Agreed. I'll try to work on this tomorrow.

**Comment by zamith** [comment]
Closed in favor of #6. Thanks for the PR. 👍

**Pull request closed without merging**